configs/components/oasis3mct/oasis3mct.yaml

```diff
@@ -5,6 +5,7 @@ time_step: 900 # changed by coupled setup anyway
 norestart: F
 use_lucia: False
 mct_version: "2.8" # TODO: set mct_version = version (can't do at the moment as version is a string)
+git-repository: https://gitlab.dkrz.de/modular_esm/oasis3-mct.git
```

Comment:
Moved it here because it is more clear and friendlier to keep the default version and the repository together as they are closely related.